setup.py

```diff
@@ -58,6 +58,22 @@ def find_version(*file_paths):
     "requests",
     "ldap3",
 ]
+EXTRAS_REQUIRE = {
+    'docs': [
+        'sphinx',
+        'sphinx_rtd_theme',
+        'sphinxcontrib-httpdomain',
+        'hieroglyph',
+        'sphinxcontrib-svg2pdfconverter',
+        (
+            'cylc-sphinx-extensions'
+            ' @ '
+            'git+https://github.com/cylc/cylc-sphinx-extensions.git'
+            '#egg=metomi_rose[all]'
```

Comment:
Install direct from GitHub for now, see also https://github.com/cylc/cylc-sphinx-extensions/issues/3